doc/format.md

```diff
@@ -288,20 +356,48 @@ If only one of these features is found in a function, the rule will not match.
 
 # extracted features
 
-capa extracts features from multiple scopes, starting with the most specific (instruction) and working towards the most general:
+capa matches features at multiple scopes, starting small (e.g., instruction) and growing large (e.g., file). In static analysis, scopes grow from instruction, to basic block, function, and then file. In dynamic analysis, scopes from call, to thread, process, and then to file:
 
 | scope       | best for...                                                                              |
 |-------------|------------------------------------------------------------------------------------------|
+| (static)    | ---                                                                                      |
 | instruction | specific combinations of mnemonics, operands, constants, etc. to find magic values       |
 | basic block | closely related instructions, such as structure access or function call arguments        |
 | function    | collections of API calls, constants, etc. that suggest complete capabilities             |
+| (dynamic)   | ---                                                                                      |
+| call        | single API call and its arguments                                                        |
+| thread      | sequence of related API calls                                                            |
+| process     | combinations of other capabilities found within a (potentially multi-threaded) program   |
+| (common)    | ---                                                                                      |
 | file        | high level conclusions, like encryptor, backdoor, or statically linked with some library |
-| (global)    | the features available at every scope, like arch or OS                                   |
+| global      | the features available at every scope, like arch or OS                                   |
 
 In general, capa collects and merges the features from lower scopes into higher scopes;
 for example, features extracted from individual instructions are merged into the function scope that contains the instructions.
 This way, you can use the match results against instructions ("the constant X is for crypto algorithm Y") to recognize function-level capabilities ("crypto function Z").
 
+| feature                           | static scope | dynamic scope |
```

Comment:
can we better indicate that this is the lowest available scope? e.g. in the text or in the headings?

Reply:
![image](https://github.com/mandiant/capa-rules/assets/156560/08ab6787-c054-4d79-8c84-f9cb71323b3f)

better? worse?